tests/tensorflow2/test_keras_mirrored.py

```diff
@@ -124,6 +124,7 @@ def scale(image, label):
         steps = ["train"]
     for step in steps:
         if step == "train":
+            hook.save_scalar("epoch", 1, sm_metric=True)
```

Comment:
can you verify that the scalar has been written using tests.core.utils.verify_files? at least in one test?

Reply:
also add an example to examples/tensorflow2 covering this case?

Reply:
> can you verify that the scalar has been written using tests.core.utils.verify_files? at least in one test?

Done